dashboard/requirements.txt

```diff
@@ -3,6 +3,9 @@ pylint
 altair
 streamlit
 streamlit_autorefresh
+matplotlib
+numpy
 requests
 python-dotenv
-psycopg2-binary
\ No newline at end of file
+psycopg2-binary
+
```

Comment:
This is a bigger list of dependencies but this is understandable for a dashboarding script.